pkg/orgIdColumnPopulator/metrics.go

```diff
@@ -0,0 +1,52 @@
+package orgIdColumnPopulator
+
+import (
+	"log"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/push"
+)
+
+type MetricsRecorder func(tableName string, uniqueAccounts int64, rowsUpdated int64) error
+
+func FakeMetricsRecorder() (MetricsRecorder, error) {
+	return func(tableName string, uniqueAccounts int64, rowsUpdated int64) error {
+		log.Println("METRICS:")
+		log.Println("tableName: ", tableName)
+		log.Println("uniqueAccounts:", uniqueAccounts)
+		log.Println("rowsUpdated:", rowsUpdated)
+		return nil
+	}, nil
+}
+
+func PrometheusMetricsRecorder(pushGateway string) (MetricsRecorder, error) {
+	return func(tableName string, uniqueAccounts int64, rowsUpdated int64) error {
+		log.Println("METRICS:")
+		log.Println("pushGateway: ", pushGateway)
+		log.Println("tableName: ", tableName)
+		log.Println("uniqueAccounts:", uniqueAccounts)
+		log.Println("rowsUpdated:", rowsUpdated)
+
+		uniqueAccountsMetric := prometheus.NewCounterVec(prometheus.CounterOpts{
+			Name: "org_id_column_populator_unique_accounts",
+			Help: "Number unique accounts processed per table.",
+		}, []string{"table"})
+		rowsUpdatedMetric := prometheus.NewCounterVec(prometheus.CounterOpts{
+			Name: "org_id_column_populator_rows_updated",
+			Help: "Number rows updated per table.",
+		}, []string{"table"})
+
+		uniqueAccountsMetric.With(prometheus.Labels{"table": tableName}).Add(float64(uniqueAccounts))
+		rowsUpdatedMetric.With(prometheus.Labels{"table": tableName}).Add(float64(rowsUpdated))
+
+		err := push.New(pushGateway, "org_id_column_populator_job").
```

Comment:
job name should be unique (e.g. host/pod name) otherwise it won't be possible to distinguish metrics from different runs

Reply:
Fixed.